main/WebServer.cpp

```diff
@@ -1280,7 +1280,38 @@ namespace http {
 				mode2 = 500;
 			}
 
-			if (htype == HTYPE_PythonPlugin) {
+			if (htype == HTYPE_HTTPPOLLER) {
+				std::string mode1Str = request::findValue(&req, "Mode1");
+				std::string mode2Str = CURLEncode::URLDecode(request::findValue(&req, "Mode2"));
+				std::string mode3Str = CURLEncode::URLDecode(request::findValue(&req, "Mode3"));
+				std::string mode4Str = CURLEncode::URLDecode(request::findValue(&req, "Mode4"));
+				std::string mode5Str = CURLEncode::URLDecode(request::findValue(&req, "Mode5"));
+				std::string mode6Str = CURLEncode::URLDecode(request::findValue(&req, "Mode6"));
+				std::string szScriptEncoded = base64_encode((const unsigned char *)extra.c_str(), extra.length());
+				std::string szMethodEncoded = base64_encode((const unsigned char *)mode1Str.c_str(), mode1Str.length());
```

Comment:
Well the above was just pseudo code
But they exists, have a look at the btoa() and atob() functions